llvm/lib/Transforms/Utils/SSAUpdaterBulk.cpp

```diff
@@ -223,3 +224,78 @@ void SSAUpdaterBulk::RewriteAllUses(DominatorTree *DT,
     }
   }
 }
+
+bool SSAUpdaterBulk::simplifyPass(SmallVectorImpl<PHINode *> &Worklist) {
+  if (Worklist.empty())
+    return false;
+
+  const DataLayout &DL = Worklist.front()->getParent()->getDataLayout();
+  bool Change = false;
```

Comment:
```suggestion
  bool Changed = false;
```
nit: More typical naming for this flag.